.gitignore

```diff
@@ -48,3 +48,9 @@ public/__sitemaps/
 
 #A spot for developers to put miscellaneous scripts.
 /personal_bin/*
+
+
+bin/oh_microsite_export/data
+bin/oh_microsite_export/local_database_name.txt
+bin/oh_microsite_export/local_database_password.txt
+bin/oh_microsite_export/local_database_user.txt
```

Comment:
you can just ignore `bin/oh_microsite_export/*` if you want to ignore everything in that dir?

Also, these files seem to have been added to the commit after all?

Reply:
Fixed.